netzob/src/netzob/Model/Protocol.py

```diff
@@ -61,45 +62,69 @@ class Protocol(object):
 
     """
 
-    def __init__(self, name=None, format_zdl=None, automata_zdl=None):
+    # Constants
+    SYMBOLS = 0
+    AUTOMATA = 1
+
+    # Static variables
+    definitions = {}
+
+    def __init__(self, name, path_zdl=None):
```

Comment:
Can you describe the objectives and the use cases of a « static list of protocols definitions »